README.md

```diff
@@ -402,6 +402,27 @@ aa('viewedFilters', {
 | `eventName` | `string`   | The name of the event                                            |
 | `filters`   | `string[]` | The list of filters that was clicked as `'${attr}${op}${value}'` |
 
+### Batch Events
+
+You can send multiple events in a single HTTP request, by using `sendEvents` method.
+
+```js
+aa('sendEvents', [
+  {
+    eventType,
+    eventName,
+    userToken,
+    ...
```

Comment:
How come we don't document other parameters?

Reply:
it all depends on what kind of event, like if it's a filter, then `filters` should be there. if it's a click, then objectID should be there, etc.